packages/app/src/elements/app.ts

```diff
@@ -156,6 +156,7 @@ export class App extends ServiceWorker(StateMixin(AutoSync(ErrorHandling(AutoLoc
                 display: flex;
                 flex-direction: column;
                 background: var(--app-backdrop-background);
+                letter-spacing: var(--letter-spacing);
```

Comment:
This is just a minor change, but can help to make the UI look a tiny bit less dense which is easier on the eye.